tests/wpt/mozilla/tests/mozilla/webgl/getFramebufferAttachmentParameter.html

```diff
@@ -0,0 +1,114 @@
+    <!doctype html>
```

Comment:
Nit: spaces before the doctype.

Reply:
Indeed, thanks. Fixing in next iteration.